src/navigation/components/Navigation/Navigation.tsx

```diff
@@ -43,6 +49,8 @@ export const Navigation: React.FC<NavigationProps> = ({data, logo, className}) =
     const onSidebarOpenedChange = (isOpen: boolean) => setIsSidebarOpened(isOpen);
 
     useEffect(() => {
+        if (!withBorderOnScroll) return () => {};
```

Comment:
I think we are no need new props to solve your problem from issue

Reply:
This PR is not related to the issue. Its two independents problems

Border no needed at all, like on this landing (css trick now)
https://ytsaurus.tech/

Reply:
ok